### PR TITLE
(docs) add distributed / ray api docs

### PR DESCRIPTION
**Pull request by kukushking**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubStandardCodeBuild8C06-llutOAimTATs
* Commit ID: 44ff6858b504fc6340b98454babee63c542089ed
* Result: FAILED
* [Build Logs](https://jsv9u8gote.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=d3c9a4dc-ede9-479b-b578-5578c6299bfa%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubDistributedCodeBuild6-jWcl5DLmvupS
* Commit ID: 44ff6858b504fc6340b98454babee63c542089ed
* Result: FAILED
* [Build Logs](https://p0rshq9q5i.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e86f01e8-f910-4a8b-b086-54de0a1018c2%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubLoadTests5656BB24-s6u9F3qN9oFy
* Commit ID: 44ff6858b504fc6340b98454babee63c542089ed
* Result: SUCCEEDED
* [Build Logs](https://pciyom2pbh.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=b9534978-6130-448d-835d-3fc570015b86%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: docs/source/api.rst [hunk 1]
```diff
@@ -429,3 +430,17 @@ Global Configurations
...
+.. currentmodule:: awswrangler.distributed.ray
+
+.. autosummary::
+    :toctree: stubs
+
+    RayLogger
+    initialize_ray
+    ray_get
+    ray_logger
+    ray_remote
```
[the remark this answers is not in the document]
Comment (kukushking) [reply docs/source/api.rst]: Done

**Review by jaidisido — APPROVED** [review]

**Pull request merged**